src/containers/LoginPage/index.js

```diff
@@ -0,0 +1,137 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import { connect } from "react-redux";
+import { login } from "../../action/login"
+import { routes } from "../Router";
+import { push } from "connected-react-router";
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Link from '@material-ui/core/Link';
+import Grid from '@material-ui/core/Grid';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import { CssBaseline } from "@material-ui/core";
+
+const Root = styled.div`
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const Banner = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+`
+
+const Image = styled.img`
+  width: 50%;
+  border-radius: 50%;
+  margin-bottom: 15px;
+`
+
+const Form = styled.form`
+  margin: 2vh 0;
+`
+
+class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+    }
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { email, password } = this.state
+    this.props.login(email, password)
+  }
+
+  render() {
+
+    return (
+      <Root>
+        <Container component="main" maxWidth="xs">
+          <CssBaseline />
+          <div>
+            <Banner>
+              <Image src={require('../../logo.png')} />
```

Comment:
É preferível que esse require esteja no início do arquivo, sendo atribuído a uma variável que ai sim seria passada aqui no `src`. Isso é um detalhe de performance e não foi explicado, mas acho legal apontar. 
O problema é que a função `require` é custosa e se está aqui no meio do componente será executada a cada renderização dele, enquanto só precisamos disso uma vez.